openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -256,6 +262,10 @@ OpenAssessment.ResponseView.prototype = {
         return this.baseView.buttonEnabled('.submission__save', enabled);
     },
 
+    uploadEnabled: function(enabled) {
```

Comment:
nit: the name is off putting to me because we are use this function to toggle functionality. I would suggest 'toggleEnableUpload' .

Reply:
This function can act as a getter or a setter, depending on the number of args provided, much like `saveEnabled()` above (or lots of other JS functions in this project) - 
```js
view.uploadEnabled(true); // enables the upload button.
view.uploadEnabled(); // checks whether the upload button is enabled.
```